Assets/Scripts/EnemySpawn.cs

```diff
@@ -44,20 +45,18 @@ public class EnemySpawn : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
+        pum = GameObject.FindGameObjectWithTag("Player").GetComponent<PowerUpManager>();
         waves = new Wave[totalWaveNumber];
-        string[] enemy = new string[1];
-        enemy[0] = "Enemy";
+        EnemyType[] enemy = new EnemyType[2];
+        enemy[0] = EnemyType.TURRET;
+        enemy[1] = EnemyType.GRENADIER;

```

Comment:
Why are we doing this again ?

Reply:
I am hard coding some parts when initializing the wave at the moment. Once we implement the enemy then the wave will be initialized differently. Will refactor after tomorrow's presentation